js/index.js

```diff
@@ -93,23 +93,28 @@ messageForm.addEventListener("submit", function (event) {
   messageForm.reset();
 });
 
-// Github request
-const githubRequest = new XMLHttpRequest();
-githubRequest.open("GET", "https://api.github.com/users/CristianMoran1/repos");
-
-githubRequest.addEventListener("load", function (event) {
-  const repositories = JSON.parse(githubRequest.responseText);
-  console.log(repositories);
-
-  // Display repositories in list
-  const projectSection = document.getElementById("projects");
-  const projectList = projectSection.querySelector("ul");
-
-  for (let i = 0; i < repositories.length; i++) {
-    const project = document.createElement("li");
-    project.innerText = repositories[i].name;
-    projectList.appendChild(project);
-  }
-});
-
-githubRequest.send();
+//Using the Fetch API, create a "GET" request to the same GitHub API url as before
+fetch("https://api.github.com/users/CristianMoran1/repos", {
+  method: "GET",
+})
+  .then(function (response) {
+    return response.json();
+  })
+  .then(function (repositories) {
+    // Display repositories in list
+    const projectSection = document.getElementById("projects");
+    const projectList = projectSection.querySelector("ul");
+
+    for (let i = 0; i < repositories.length; i++) {
+      const project = document.createElement("li");
+      project.innerText = repositories[i].name;
+
+      project.addEventListener("click", function () {
+        window.open(repositories[i].html_url, "_blank");
```

Comment:
great job!

Reply:
another way to do these links would be to wrap the contents of the li inside of one big `<a>` tag